src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Color {
```

Comment:
It's good practice to add access modifiers to your enums. In this case, it would be 'public enum Color' which is already correct, so no change is needed here.